tests/test_migrations.py

```diff
@@ -28,3 +35,99 @@ def test_for_missing_migrations(self):
 
         if status_code == '1' and "djangocms_link" in output:
             self.fail(f'There are missing migrations:\n {output.getvalue()}')
+
+@skipIf(
+    __version__ >= '5' or cms_version < "4",
```

Comment:
```suggestion
    __version__ > '5' or cms_version < "4",
```
Do not skip this test for version 5